nengo_ocl/simulator.py

```diff
@@ -662,14 +662,13 @@ def plan_ElementwiseInc(self, ops):
 
     def plan_SimPyFunc(self, ops):
         groups = groupby(ops, lambda op: op.fn)
-        plans = []
-        for fn, group in groups:
-            assert all(op.fn == fn for op in group)
-            plans.extend(self._plan_python_fn(fn,
-                                              [op.t for op in group],
-                                              [op.x for op in group],
-                                              [op.output for op in group]))
-        return plans
+        # ^ NOTE: Groups functions based on equality `==`, not identity `is`.
+        #   I think this is what we want in all cases.
+        return [
```

Comment:
Not sure if it matters, but I think this list comprehension is slightly different from what this did previously. `_plan_python_fn` returns a list; previously, all the returned lists would be added to a single `plans` list. Now, with the list comprehension, you get a list of lists. Maybe this is handled elsewhere so it doesn't matter, but just pointing it out in case it matters :)